src/Administration/Resources/app/administration/src/app/assets/scss/global.scss

```diff
@@ -158,3 +158,8 @@ a[target="_blank"]:not(.sw-external-link, .sw-internal-link, .mt-button, .mt-lin
 .mt-inheritance-switch {
     margin-right: 8px;
 }
+
+// TODO: remove when the list style type issue is fixed in the tooltip content
```

Comment:
```suggestion
```
Either link a ticket for that or just leave it out

Reply:
Does it look very bad without this? 🤔 I'm thinking that the Meteor Component will be fixed anyway soon, which would be better than introducing soon-to-be dead code, you know

@Haberkamp you think the PR can be merged soon? 😄